agent/websockets/connection.go

```diff
@@ -208,20 +227,20 @@ func (conn *Connection) SendClientMessage(msg interface{}) error {
 //
 // The returned []string value is nil if the error is non-nil, or if the method
 // times out while waiting for a server message.
-func (conn *Connection) ReadServerMessages() ([]interface{}, error) {
+func (conn *Connection) ReadServerMessages(protocolVersion int) ([]interface{}, error) {
```

Comment:
This new `protocolVersion` method argument does not seem necessary

Reply:
Merge issues as mentioned above.